db/migrate/20210216113138_change_admin_team_type_to_other.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAdminTeamTypeToOther < ActiveRecord::Migration[6.0]
```

Comment:
I think once this is all done in production, we should have a PR to remove this. Rails migration tend to be structurally changes only but I think this is easy and work